sql/src/test/java/io/crate/integrationtests/JoinIntegrationTest.java

```diff
@@ -969,7 +969,7 @@ public void test_block_NestedLoop_or_HashJoin__with_group_by_on_right_side() {
         CircuitBreaker queryCircuitBreaker = internalCluster().getInstance(CrateCircuitBreakerService.class).getBreaker(CrateCircuitBreakerService.QUERY);
         randomiseAndConfigureJoinBlockSize("t1", 10L, queryCircuitBreaker);
 
-        execute("select x from t1 left_rel JOIN (select x x2, count(x) from t1 group by x2) right_rel " +
```

Comment:
Not sure about this one. If i leave it to x, and `AmbiguousColumnException` is thrown.

Reply:
This should work because the right relation shadows `x` with `x2`, so the `x` in the select-list isn't ambiguous